src/index.js

```diff
@@ -1 +1,19 @@
-'use strict';

const scale = require('./mode');
const chord = require('./chord');
const clip = require('./clip');
const pattern = require('./pattern');
const midi = require('./midi');
const scales = require('./modes');

let modes = Object.keys(scales);

// Allow scale to be denoted by mode as well
module.exports = {mode: scale, scale, chord: chord.getChord, listChords: chord.listChords, modes, scales: modes, clip, pattern, midi};
\ No newline at end of file
+'use strict';
+const scale = require('./mode');
+const chord = require('./chord');
+const clip = require('./clip');
+const pattern = require('./pattern');
+const midi = require('./midi');
+const scales = require('./modes');
+function setMiddleC(octaveIndex){
+    if(typeof octaveIndex !== "number" || Math.floor(octaveIndex) !== octaveIndex){
```

Comment:
This check should be in the main function (which is in the midi.js file).
Also here you can omit defining this function. Instead later in this file where we have `module.exports = ...`, could you check if something like this would work:
```
module.exports = {mode: scale, scale, chord: chord.getChord, listChords: chord.listChords, modes, scales: modes, clip, pattern, midi: midi.midi, setMiddleC: midi.setMiddleC};
```